### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by LappleApple**
Went ahead and did it :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Pull requests only
+
+**DON'T** push to the master branch directly. Always use feature branches and let people discuss changes in pull requests.
+Pull requests should only be merged after all discussions have been concluded and at least 1 reviewer has given their 
+**approval**.
+
+## Guidelines
+
+- **every change** needs a test
+- 100% code coverage
```
Comment (Raffo) [inline CONTRIBUTING.md]: 100% code coverage is a bit unreasonable most of the times, I would remove this guideline.

Comment (LappleApple) [inline CONTRIBUTING.md]: OK!

**Comment by Raffo** [comment]
LGTM.

**Comment by aryszka** [comment]
Nice! Thanks a lot! :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+## Pull requests only
+
+**DON'T** push to the master branch directly. Always use feature branches and let people discuss changes in pull requests.
+Pull requests should only be merged after all discussions have been concluded and at least 1 reviewer has given their 
+**approval**.
+
+## Guidelines
+
+- **every change** needs a test
+- keep the current code style
```
Comment (szuecs) [inline CONTRIBUTING.md]: Does this "current code style" apply to not "fixing" go lint style issues?

Comment (lmineiro) [inline CONTRIBUTING.md]: This has been discussed before. If "fixing" go lint means breaking the API, then, yes, this means not to fix go lint. Any other, non-breaking, "go lint fixes" are welcome

Comment (aryszka) [inline CONTRIBUTING.md]: we could pharse it somehow so: 1. use go fmt, 2. use golint for new changes, not breaking backwards compatibility

Comment (lmineiro) [inline CONTRIBUTING.md]: :+1:

**Pull request merged**